basics/basic-types.md

```diff
@@ -0,0 +1,118 @@
+# Elementare Datentypen
+
+D bietet eine Vielzahl an elementaren Datentypen, die immer die gleiche Größe haben -
+**unabhängig** von der Platform.
+Die einzige Ausnahme ist der Typ `real` für welchen der Compiler
+die höchstmöglichste Genauigkeit für Gleitkommazahlen wählt.
+Es besteht kein Unterschied zwischen der Größe eines Ganzzahltypen - unabhängig davon
+ob die Anwendung für eine 32-bit oder 64-bit Architektur kompiliert wird.
+
+<table class="table table-hover">
+<tr><td width="250px"><code class="prettyprint">bool</code></td> <td>8-bit</td></tr>
+<tr><td><code class="prettyprint">byte, ubyte, char</code></td> <td>8-bit</td></tr>
+<tr><td><code class="prettyprint">short, ushort, wchar</code></td> <td>16-bit</td></tr>
+<tr><td><code class="prettyprint">int, uint, dchar</code></td> <td>32-bit</td></tr>
+<tr><td><code class="prettyprint">long, ulong</code></td> <td>64-bit</td></tr>
+</table>
+
+#### Gleitkommatypen
+
+<table class="table table-hover">
+<tr><td width="250px"><code class="prettyprint">float</code></td> <td>32-bit</td></tr>
+<tr><td><code class="prettyprint">double</code></td> <td>64-bit</td></tr>
+<tr><td><code class="prettyprint">real</code></td> <td>abhängig von der Platform, 80-bit für Intel x86 32-bit</td></tr>
+</table>
+
+Der Präfix `u` kennzeichnet Typen ohne Vorzeichen (vom Englischen `unsigned`).
+Ein `char` ist ein UTF-8 Zeichen, `wchar` ein UTF-16 Zeichen and `dchar`
```

Comment:
> Is this accurate though? 

I translated this literally:

> The prefix u denotes unsigned types. char translates to UTF-8 characters, wchar is used in UTF-16 strings and dchar in UTF-32 strings.

https://github.com/dlang-tour/english/blob/master/basics/basic-types.md

>  A char is only 1 byte and represents a byte in a UTF-8 string, you might need multiple chars to represent 1 UTF-8 character. You have dchar for full characters without needing to have multiple of them.

The document is referring to code units here, e.g. from Wikipedia:

https://en.wikipedia.org/wiki/UTF-8
> The encoding is variable-length and uses 8-bit code units. It was designed for backward compatibility with ASCII and to avoid the complications of endianness and byte order marks in the alternative UTF-16 and UTF-32 encodings
https://en.wikipedia.org/wiki/UTF-16
> The encoding is variable-length, as code points are encoded with one or two 16-bit code units.
https://en.wikipedia.org/wiki/UTF-32
>  It is a protocol to encode Unicode code points that uses exactly 32 bits per Unicode code point.

Maybe we should edit the base document to make it a bit clearer that code units are referred to?
FYI it's explained in more details on [the strings page](http://tour.dlang.io/tour/en/basics/alias-strings) and  since yesterday the DTour has a new gem on [Unicode](http://tour.dlang.io/tour/en/gems/unicode)